### PR TITLE
Fixes #2 and some other stuff

### LINKED ISSUE #2: Add support for automatically adding new license contacts to MailChimp list
It would be great to be able to make filters like on the License Report page and automatically add contacts from those filter results to certain distribution lists on a linked MailChimp account.

### PR DESCRIPTION
**Pull request by spraot**
Here's a few improvements if you are interested.

### CODE REVIEW

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -1,4 +1,6 @@
 parameters:
+    base_url: http://localhost:8000
```
Comment (thecatontheflat) [inline app/config/parameters.yml.dist:2]: :heart_eyes_cat: :+1:

File: src/AppBundle/Service/AddLicenseContact.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        for ($i = 0; $i < count($listMapTmp); $i++) {
+            $listMaps[$i] = explode(':',$listMapTmp[$i]);
+
+            if (count($listMaps[$i]) != 2) {
+                throw new Exception('Badly formed list_map in parameters.yml.');
+            }
+        }
+
+        $this->listMap = $listMaps;
```
Comment (thecatontheflat) [inline src/AppBundle/Service/AddLicenseContact.php:36]: Could you please describe the logic here? Is it necessary to have this parameter in such format `plugin:list_id;plugin2:list_id2`? I would use YAML array.

**Comment by thecatontheflat** [comment]
I appreciate your PR and would like to merge it.
Could you please clarify some points mentioned above? Also, is it possible to create some sort of a feature flag for people, who don't have mailchimp?
Example of existing feature flag:
https://github.com/thecatontheflat/atlassian-vendor-tools/blob/master/app/config/parameters.yml.dist#L24
https://github.com/thecatontheflat/atlassian-vendor-tools/blob/master/src/AppBundle/Command/ImportSaleCommand.php#L91-L93

**Comment by thecatontheflat** [comment]
I am merging the PR. If you feel like implementing fixes for the feedback — would be awesome. Otherwise I'll just figure it out later during this week.

Thanks @rachetfoot !

File: src/AppBundle/Service/AddLicenseContact.php (hunk 2 above)
Comment (spraot) [inline src/AppBundle/Service/AddLicenseContact.php:36]: Absolutely not necessary, I'm just not familiar with YAML or PHP.

What happens there is first getting the datacenter id from API key, then seperating the list of plugin key / list IDs. When there is a new license for a given plugin key, the technical contact will be added to the mailchimp list with that ID.

**Comment by spraot** [comment]
By the way, I left out the feature flag because the same is accomplished by leaving the listMap string empty.

**Comment by thecatontheflat** [comment]
@rachetfoot ah, makes sense.

**Comment by spraot** [comment]
I think that's a good idea. I wasn't sure if there were any limitations on the way the YAML gets marshalled to a php variable, so I just did what I knew would work.

File: src/AppBundle/Service/AddLicenseContact.php (hunk 2 above)
Comment (thecatontheflat) [inline src/AppBundle/Service/AddLicenseContact.php:36]: Thanks! I'll adapt the implementation according to the framework guidelines

**Pull request merged**